Runtime/Tracking/Collision/CollisionNotifier.cs

```diff
@@ -167,8 +167,8 @@ public enum CollisionTypes
         /// <returns><see langword="true"/> if events should be emitted.</returns>
         protected virtual bool CanEmit(EventData data)
         {
-            return (data.isTrigger && emittedTypes.HasFlag(CollisionTypes.Trigger)
-                    || !data.isTrigger && emittedTypes.HasFlag(CollisionTypes.Collision))
+            return (data.isTrigger && (emittedTypes & CollisionTypes.Trigger) != 0

```

Comment:
should we be explicit here and do `emittedTypes != null`?

I know it's not "needed" but in some places you do need to be explicit (cos Unity) and at least the code then looks consistent

Reply:
I agree, but in this case this is doing a bitwise check on `emittedTypes` - the actual expression is `(emittedTypes & CollisionTypes.Trigger) != 0`. It's not doing a logical AND, it's a bitwise AND :)

Reply:
Oh yeah I didn't see the single & 😄